lib/nimble_template/addons/github.ex

```diff
@@ -51,4 +51,14 @@ defmodule NimbleTemplate.Addons.Github do
 
     project
   end
+
+  @impl true
+  def do_apply(%Project{} = project, opts) when is_map_key(opts, :github_wiki) do
+    template_file_path = ".github/workflows/publish_wiki.yml.eex"
+    file_path = ".github/workflows/publish_wiki.yml"
+
+    Generator.copy_file([{:text, template_file_path, file_path}])
```

Comment:
should we create the `.github/wiki` folder, along with some initial files like 

- Home
- _Sidebar
- Getting-Started

Reply:
Added in a111487b059b19f8586a303a42d561d67c1d168e